apps/src/templates/teacherDashboard/SectionTable.story.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import SectionTable from './SectionTable';
+
+const sections = [
+  {
+    id: 11,
+    name: "brent_section",
+    loginType: "word",
+    grade: null,
+    stageExtras: false,
+    pairingAllowed: true,
+    numStudents: 10,
+    code: "PMTKVH",
+    assignmentName: "CS Discoveries",
+    assignmentPath: "//localhost-studio.code.org:3000/courses/csd"
```

Comment:
Why is that problematic? This is in a .story file where localhost makes more sense to me than some other env.